internal/utils_test.go

```diff
@@ -1262,6 +1262,74 @@ func TestGetPolicyTemplateInvalidManifest(t *testing.T) {
 	assertEqual(t, err.Error(), expected)
 }
 
+func TestGetPolicyTemplateObjectTemplatesRaw(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	manifestPath := path.Join(tmpDir, "configmap.yaml")
+	manifestYAML := `
+object-templates-raw: |
```

Comment:
Could you have a test to verify when a single manifest file has two object-templates-raw fields? For example:

```yaml
object-templates-raw: |
  something
----
object-templates-raw: |
  something2
```

Reply:
I ended up just making the test have two `object-templates-raw` fields instead of one. Would it still be necessary to test the singular case?

Reply:
I think one test with both in it is fine.